packages/functional-tests/pages/cookiesDisabled.ts

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { BaseLayout } from './layout';
+
+export class CookiesDisabledPage extends BaseLayout {
+  readonly path = 'cookies_disabled';
+
+  async isCookiesDisabledHeader() {
+    return this.page.locator('.card-header').isVisible();
```

Comment:
Yeah as long as we keep POM model intact with page class- selectors/ functions separate than the actual tests we will be good even if we use same kind of class names in different pages. But in case later we see some interference we can switch back to `getByText` or `getByRole`.